ansible/roles/dns/README.md

```diff
@@ -0,0 +1,43 @@
+dns
```

Comment:
This role should stay in openshift-ansible, it is used by openshift-on-openstack and will be used by the Cloud Forms integration.